.env.example

```diff
@@ -1,6 +1,7 @@
 INFURA_PROJECT_ID=
 DEPLOYMENT_ACCOUNT_PK=
 GAS_PRICE=21000000000
-KYBER_DAO_ADDRESS=0x806f978fec453161d11c390c76edb78ff3a010be
+KYBER_DAO_ADDRESS=0x98fac5AD613c707Ef3434B609A945986e4d05d07
+KYBER_FEE_HANDLER_ADDRESS=0xe57B2c3b4E44730805358131a6Fc244C57178Da7
```

Comment:
I think we should also mention the new variable in the `Deployment` section of the README file. Similar as it this now KYBER_DAO_ADDRESS with the link to https://github.com/KyberNetwork/developer-portal/blob/stakingSection/testnet.md